bundle/analyze/superset/superset-init.sh

```diff
@@ -37,52 +34,43 @@ Init Step ${1}/${STEP_CNT} [${2}] -- ${3}
 
 EOF
 }
-ADMIN_PASSWORD="admin"
-# If Cypress run – overwrite the password for admin and export env variables
-if [ "$CYPRESS_CONFIG" == "true" ]; then
-    ADMIN_PASSWORD="general"
-    export SUPERSET_CONFIG=tests.superset_test_config
-    export SUPERSET_TESTENV=true
-    export ENABLE_REACT_CRUD_VIEWS=true
-    export SUPERSET__SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://superset:superset@db:5432/superset
+
+# Install drivers
+if [ ! -z "${SUPERSET_ADDITIONAL_DEPENDENCIES}" ]; then
+  echo_step "1" "Starting" "Install additional dependencies"
+  pip install ${SUPERSET_ADDITIONAL_DEPENDENCIES}
+  echo_step "1" "Complete" "Install additional dependencies"
+else
+    echo_step "1" "Skipping" "Install additional dependencies"
 fi
+
+ADMIN_PASSWORD="admin"
 # Initialize the database
-echo_step "1" "Starting" "Applying DB migrations"
+echo_step "2" "Starting" "Applying DB migrations"
 superset db upgrade
-echo_step "1" "Complete" "Applying DB migrations"
+echo_step "2" "Complete" "Applying DB migrations"
 
 # Create an admin user
-echo_step "2" "Starting" "Setting up admin user ( admin / $ADMIN_PASSWORD )"
+echo_step "3" "Starting" "Setting up admin user ( admin / $ADMIN_PASSWORD )"
 superset fab create-admin \
```

Comment:
It doesn't make sense to create a new admin every time we start Superset, and there's currently no clear way for them to set their own password, so we'd be creating insecure installations by default. 

We should instead instruct the user to do this themselves (through `meltano invoke`) in every environment, as is currently the case with Airflow.

Do you know if this is a required step, or if Superset will also work without an admin user?

Reply:
@DouweM Yeah if the user exists already then this just logs, "users already exists" but somehow configuring their own password is a good point.

This is somewhat similar to my comment from https://github.com/meltano/hub/pull/540. I was sort of building this in a way to get users up and running as smooth as possible locally as if they were following https://superset.apache.org/docs/installation/installing-superset-from-scratch#installing-and-initializing-superset but with a single command, also assuming that production would probably look a little different anyways. For example the instructions and this script have us starting up a flask dev server. Personally I was imagining that the value Meltano added was that I could just run `meltano invoke superset:ui` and get a running instance vs having to start up the UI in the background then separately read the docs to know how to initialize it to be functional (i.e. db upgrade, init, create-admin, run). I'm open to either or both but it sounds like our expectations for this were a little different 😄 .